onnxruntime/core/providers/cpu/cpu_provider_factory.cc

```diff
@@ -3,6 +3,7 @@
 
 #include "core/providers/cpu/cpu_provider_factory.h"
 #include <atomic>
+#include "core/graph/onnx_protobuf.h"
 #include "cpu_execution_provider.h"
```

Comment:
Is this needed?

Reply:
yes, CPUExecutionProviderInfo below

---
In reply to: [397641237](https://github.com/microsoft/onnxruntime/pull/3174#discussion_r397641237) [](ancestors = 397641237)

Reply:
windows would have compile error on common/constants.h wo/ it
constexpr bool OPTIONAL = false;

---
In reply to: [397641237](https://github.com/microsoft/onnxruntime/pull/3174#discussion_r397641237) [](ancestors = 397641237)